README.md

```diff
@@ -201,32 +166,4 @@ the correct deployment source and the name of the service account, e.g.:
 
 The permissions are configured as with any other IAM Role deployment. Make sure these
 follow best practice, with each resource appropriately scoped by account ID, region,
-environment, etc.
-
-Continuous Integration
-----------------------
-
-CI is managed by GitHub actions. Each repository has its own custom CI pipeline, but
-they broadly follow a `Test > Build > Push image > Push HelmRelease`
-pattern.
-
-### Images
-
-Docker images are built by CI and pushed to AWS ECR. Each repository should have a
-container repository with the same name in ECR. This repository should be checked
-and created by CI if it is not present.
-
-### Deployment
-
-Each repository is responsible for managing its own HelmRelease files. Each repository
-that deploys a HelmRelease should have a `.flux.ci` file that contains the base template
-that the CI pipeline will fill in and extend. The CI pipeline for a given project will
-push this filled template to this repository under `releases/` for Flux to pick it up.
-
-### CI privileges
-
-Each CI pipeline needs a certain set of permissions from both GitHub and AWS to operate.
-AWS privileges that should be available to the CI pipeline should be placed under `.ci.yaml`.
-
-Changes to CI privileges must be deployed *manually* by someone with sufficient access
-by running `biomage rotate-ci` using [biomage-utils](https://github.com/hms-dbmi-cellenics/biomage-utils).
```

Comment:
We definitely need this text somewhere. Are you planning to move it to `releases`?